PythonPonies/bangazon-api
# admin tables now read with correct values

# Description
Add description about what this pull request covers.
This applies a fix to properly display the correct fields for each of the models/tables in the admin panel. The panels should now read with actual names of the users on the user model and similarly for each of the tables. 
## Number of Fixes
Type of number of fixes included in this PR.
Several iterations of the fix: creating a method on the model to return the appropriate fields as a string. 

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
No related tickets. 

## Problem to Solve
Describe the problem(s) this pull request solves.
Provides better clarity on what the individual objects relate to in each of the associated models in the admin user panel. 

## Proposed Changes
Describe the proposed changes.
We'll need to fix the indentation error that pops up when we try to use add a similar method to the payment_type model. 

## Expected Behavior
Describe the expected behavior.
You should see names in the user panel, titles in the products panel, order id and buyer in the order panel. 

## Steps to Test Solution
1. Git fetch this branch
2. Checkout branch
3. Run the server
4. Navigate to localhost:8000/admin
5. Click through each of the tables and check that appropriate data is there. 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
